src/types/common/enums/common.enums.ts

```diff
@@ -32,7 +32,10 @@ export enum VariantEnum {
   Tonal = 'tonal',
   Standard = 'standard',
   Caption = 'caption',
-  Button = 'button'
+  Button = 'button',
+  H5 = 'h5',
```

Comment:
this variant enum should be separated because we have here typography variants style variants, variant text is not used at all idk why we need it

Reply:
Ok, I agree with you. Then we should create an interface for each component because I looked and saw that they are different.

Reply:
I deleted it because it doesn't need me. But it will need to fix